third_party/blink/renderer/modules/cobalt/h5vcc_system/h_5_vcc_system.h

```diff
@@ -66,6 +66,11 @@ class MODULES_EXPORT H5vccSystem final
   void EnsureReceiverIsBound();
   HeapMojoRemote<h5vcc_system::mojom::blink::H5vccSystem> remote_h5vcc_system_;
 
+  // Holds promises associated with outstanding async remote_h5vcc_system_
```

Comment:
nit: `|remote_h5vcc_system_|` or `` `remote_h5vcc_system_` `` : escape variable
names in comments.